msu/systems/mod_settings/load.nut

```diff
@@ -28,6 +28,9 @@ includeFile("mod_settings_mod_addon.nut");
 ::MSU.UI.addOnConnectCallback(::MSU.System.ModSettings.finalize.bindenv(::MSU.System.ModSettings));
 ::MSU.System.ModSettings.Screen = ::MSU.SettingsScreen;
 
+::MSU.System.Serialization.registerSerializeFunction(::MSU.System.ModSettings.flagSerialize.bindenv(::MSU.System.ModSettings));
```

Comment:
I'd like to discuss the utility of these functions again but that is more like a re-discussion about PR #157